zstacklib/zstacklib/utils/linux.py

```
@@ -2523,12 +2532,18 @@ def check_socket_available(host, port, timeout=10):
     return False
 
 def is_port_available(port):
-    with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+    with contextlib.closing(socket.socket(socket.AF_INET6, socket.SOCK_STREAM)) as s:
         try:
-            s.bind(('', int(port)))
+            s.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
+            s.bind(('::', int(port)))
             return True
         except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)